restcomm/restcomm.rvd/src/main/webapp/languages/es.json

```diff
@@ -125,11 +126,11 @@
 	"dialStepRecordingConferenctWarning":"La grabación de la sesión aún no es permitida",
 
 	"downloadApplicationPackageLabel": "Descargue el paquete de aplicación para",
-	"downloadNotAvailableLabel": "Lo sentimos, su paquete de aplicación no esta habilitado",
+	"downloadNotAvailableLabel": "Lo sentimos, su paquete de aplicación no esta disponible",
 	"downloadIsAvailableLabel1": "Su paquete binario de aplicación esta habilitado. Click",
 	"downloadIsAvailableLabel2": "aquí",
 	"downloadIsAvailableLabel3": "para descargar.",
-	"downloadContinueLabel1": "Puede continuar con",
+	"downloadContinueLabel1": "Puede continuar",
 	"downloadContinueLabel2": "editando esta app en el Visual Designer",

```

Comment:
Grammar error found. Change for:
"downloadContinueLabel2": "editando esta aplicación en el Visual Designer",